### ISSUE TITLE
Docker Server ARM64/v8 not supported?

### ISSUE THREAD

**Issue by epe** [kromitgmbh/titra#206]
I get the following Error Message: 

ERROR: no matching manifest for linux/arm64/v8 in the manifest list entries

I'm using a rockchip based Linux Armbian NAS, is there just a notice needed or would it take you to re-build the whole project for that CPU type? 
Grüße aus Vorarlberg ;-)

**Comment by faburem** [comment]
I think we have to add this architecture to our build step - let me check and see what is needed to get this working!

**Comment by epe** [comment]
Hi, Danke für deine Rückmeldung! 
Kannst du einschätzen wie lange das ca. dauern würde?

**Comment by faburem** [comment]
voraussichtlich bald:
<img width="801" alt="image" src="https://github.com/kromitgmbh/titra/assets/11456790/cdac7437-8d5c-447b-a0d8-01fe9aeb1cf1">

https://guide.meteor.com/3.0-migration#When-will-Meteor-3-0-be-ready

Aber das liegt natürlich nicht in unserer Hand..

**Comment by faburem** [comment]
yeah, the main culprit here is not mongodb but the titra docker image which currently cannot be compiled for arm64 due to limitations in the Meteor.js framework. Let's hope the 3.0 release stays on track and is released soon!

**Comment by epe** [comment]
Hi Fabian, 
Yeah, my fault, I confused the both.  
I saw you have a docker image for NAS https://wiki.titra.io/books/deployment/page/titra-on-synology-nas-dsm 
what is the processor architecture on that?

LG aus dem Westen

**Comment by faburem** [comment]
The Synology NAS tutorial is using the standard docker image, there is no dedicated build necessary so I assume it is x86 or amd64

**Comment by epe** [comment]
They use all sorts of CPUs, here's a list and the Marvell ARMada is a ARMv7 Architecture.

**Comment by faburem** [comment]
There is an update on the planned release date of Meteor 3.0, I guess it might be around May:
https://github.com/meteor/meteor/discussions/12865?sort=new#discussion-5820005

**Comment by faburem** [comment]
took a bit longer than expected but arm64 support is finally here 🕺
